scripts/scil_compute_bundle_specific_fodf_metrics.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Script to compute bundle-specific fODF metrics, derived for fitting a
+Bingham distribution to each fODF lobe, as described in Riffert et al., 2014.
+Resulting metrics are fiber density (FD), fiber spread (FS) and fiber
+fraction (Schreiber et al., 2014) (FF).
+
```

Comment:
Add a newline... right now the --help print to stdout is just a big block of text

Reply:
Done in ba1e733 :+1: